src/Storage/Commands/Charge.php

```diff
@@ -89,10 +92,11 @@ public function delete(ChargeReference $chargeRef, ShopId $shopId): bool
      */
     public function makeUsage(UsageChargeTransfer $chargeObj): ChargeId
     {
+        $userTableId = Str::singular(Util::getShopifyConfig('table_names.shops', 'users')) . '_id';
```

Comment:
@nahid  This line is used in multiple spots in multiple files, I think it might be best to make a new method in Util to get this value.

Reply:
Maybe something like this:

```php
# Util file
public static getShopsTable(): string {
  return Util::getShopifyConfig('table_names.shops', 'users'));
}
```

Usage like:

```php
Util::getShopsTable()

# or

$shopsTable = Util::getShopsTable();
$userId = "{$shopsTable}_id";
$planForeign = "{$shopsTable}_plan_id";
```

Reply:
Thanks, @osiset for your reply. Yes, it would be better to reuse if we separate this code. I'll do this ASAP.